net-p2p/qbittorrent/qbittorrent-4.5.5.ebuild

```diff
@@ -19,34 +19,49 @@ fi
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="+dbus systemd +gui webui"
-REQUIRED_USE="dbus? ( gui )
-	|| ( gui webui )"
+IUSE="+dbus +gui qt6 test webui systemd"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="|| ( gui webui )"
 
 RDEPEND="
 	>=dev-libs/openssl-1.1.1:=
-	dev-qt/qtcore:5
-	dev-qt/qtnetwork:5[ssl]
-	dev-qt/qtsql:5
-	dev-qt/qtxml:5
-	<net-libs/libtorrent-rasterbar-2:=
+	>=net-libs/libtorrent-rasterbar-1.2.18:=
 	>=sys-libs/zlib-1.2.11
-	dbus? ( dev-qt/qtdbus:5 )
+	virtual/libiconv
 	gui? (
 		dev-libs/geoip
-		dev-qt/qtgui:5
-		dev-qt/qtsvg:5
-		dev-qt/qtwidgets:5
+		!qt6? (
+			dev-qt/qtgui:5
+			dev-qt/qtsvg:5
+			dev-qt/qtwidgets:5
+			dbus? ( dev-qt/qtdbus:5 )
+		)
+		qt6? (
+			dev-qt/qtbase:6[dbus?,gui,widgets]
+			dev-qt/qtsvg:6
+		)
+	)
+	qt6? ( dev-qt/qtbase:6[network,ssl,sql,sqlite,xml(+)] )
+	!qt6? (
+		dev-qt/qtcore:5
+		dev-qt/qtnetwork:5[ssl]
+		dev-qt/qtsql:5[sqlite]
+		dev-qt/qtxml:5
 	)
-
 	webui? (
 		>=acct-group/qbittorrent-0
 		>=acct-user/qbittorrent-0
 	)"
```

Comment:
I'm not sure if you want to guard this under USE="webui", personally I'd have acct-* unconditionally without IUSE control, since it's quite trivial

Reply:
fixed